modules/instruments/php/module.class.inc

```diff
@@ -59,19 +59,24 @@ class Module extends \Module
         $params    = $request->getQueryParams();
         $commentID = $params['commentID'] ?? null;
         if (empty($commentID)) {
-            return (new \Laminas\Diactoros\Response())
-                ->withBody(new \LORIS\Http\StringStream("Missing CommentID"))
-                ->withStatus(400);
+            return new \LORIS\Http\Response\JSON\BadRequest(
```

Comment:
This is not an appropriate response here, the response is not JSON, it's an error message for the client.